pkgs/servers/mastodon/update.nix

```diff
@@ -0,0 +1,39 @@
+{ lib, writeScript, fetchFromGitHub, nix-prefetch-github, yarn2nix-moretea, bundix, coreutils, jq, curl, nix, git, gawk, gnused }:
+
+let
+  # until submodules support gets merged
+  bundix_with_submodules = bundix.overrideAttrs (attrs: {
+    src = fetchFromGitHub {
+      owner = "petabyteboy";
+      repo = "bundix";
+      rev = "feature/submodules";
+      sha256 = "1xih4mz1mwba3vrynmgp6h6xnlddjjdhswx94jqrn438jj757145";
+    };
+  });
+
+in writeScript "update-mastodon" ''
```

Comment:
This is quite a long shell script to be embedded inside Nix. Consider an `update-mastodon.sh` and using `substituteAll` so it can be edited as a shell script more easily.

Reply:
I changed it to a .sh using nix-shell as shebang, which should solve this issue.